### PR TITLE
Add getAdmin in action Creator,reducer,use axios #1808

### LINKED ISSUE #1808: Add getAdmin in action Creator, reducer, use axios for api
**Actual Behaviour**

Currently admin is being set only in local react state using jquery ajax method.

**Expected Behaviour**

Integrate redux for admin state, which would allow users to use admin globally. Also, use axios for api for admin

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1808

Closed https://github.com/fossasia/chat.susi.ai/pull/1877 as the branch got old

Changes:

1. Created getAdmin function to fetch using axios.
2. Added getAdmin to Login.js and removed from StaticAppBar, TopBar
3. Created action creator and reducer for getAdmin
4. Added mockStore for enzyme for StaticAppBar

**Updated Link**: https://pr-1931-2-fossasia-susi-web-chat.surge.sh

Demo Link:  https://pr-1931-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd please review

File: src/components/Auth/Login/Login.react.js [hunk 1]
```diff
@@ -159,6 +160,26 @@ class Login extends Component {
     }
   };
 
+  callGetAdmin = accessToken => {
+    this.props.actions
```
Comment (AakashMallik) [inline src/components/Auth/Login/Login.react.js]:
```
const { actions } = this.props;
actions.getAdmin(accessToken)
```
would look better.

Comment (fragm3) [reply src/components/Auth/Login/Login.react.js]: This is also fine, as the getAdmin is not called anywhere else in the function

File: src/components/Auth/Login/Login.react.js [hunk 2]
```diff
@@ -159,6 +160,26 @@ class Login extends Component {
...
+      .getAdmin(accessToken)
+      .then(({ response }) => {
+        let { showAdmin, accepted } = response;
+        if (accepted) {
+          cookies.set('isAdmin', showAdmin, {
+            path: '/',
+            domain: cookieDomain,
+          });
+        }
+        this.setState({
```
Comment (akshatnitd) [inline src/components/Auth/Login/Login.react.js]: There is no `isAdmin` state defined. Here, call an action to set `isAdmin` to true in the Redux state.

Comment (fragm3) [reply src/components/Auth/Login/Login.react.js]: Oh, missed that. Made the changes

**Comment by fragm3** [comment]
@akshatnitd If the user has logged in, the admin cookie is set. Don't you think setting cookie once only on login is apt?

**Comment by fragm3** [comment]
@akshatnitd please review

**Comment by fragm3** [comment]
Ok, got your point, will make the changes

**Comment by akshatnitd** [comment]
Yes, we can do that now, as we are slowly integrating Redux to other web-apps as well.

**Comment by fragm3** [comment]
@akshatnitd made the changes, updated link: https://pr-1931-2-fossasia-susi-web-chat.surge.sh

**Comment by fragm3** [comment]
Will make a new Pr, with changes for issue https://github.com/fossasia/chat.susi.ai/issues/1941, together

**Pull request closed without merging**